Cue/app/actions/library.js

```diff
@@ -82,7 +83,7 @@ async function syncDeck(change) : PromiseAction {
   let serverDeck = {}
   try {
     if (change.action == "add"){
-      serverDeck = await LibraryApi.createDeck(change.name, change.tags, change.cards);
+      serverDeck = await LibraryApi.createDeck(change.name, change.tags);
```

Comment:
@lonebutterfish Just want to check if this change may affect anything of yours? Changed it because if you create a new deck with cards, you end up adding cards twice.